source/opt/ssa_rewrite_pass.cpp

```diff
@@ -507,9 +508,20 @@ bool SSARewriter::ApplyReplacements() {
   std::cerr << "\n\n";
 #endif
 
+  // Sort phi candiates by structured order
+  std::vector<const PhiCandidate*> ordered_phis_to_generate;
+  std::list<BasicBlock*> order;
+  pass_->context()->cfg()->ComputeStructuredOrder(fp, &*fp->begin(), &order);
+  for (auto block : order) {
+    for (const PhiCandidate* phi_candidate : phis_to_generate_) {
+      if (phi_candidate->bb() == block)
+        ordered_phis_to_generate.push_back(phi_candidate);
+    }
+  }
```

Comment:
@greg-lunarg Can you give explain why you need to sort the OpPhi instructions?  Would the reverse post order work?  @dnovillo Can you also double check this change?  You wrote this pass.

Reply:
Add to the comment why you are sorting.  That will be useful.